ClassHomework/src/APCSWork/Triangle.java

```diff
@@ -0,0 +1,68 @@
+package APCSWork;
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon{
+	public Triangle() {
+		super(3, new ArrayList<Point>());
+	}
+	public Triangle(ArrayList<Point> coors) {
+		super(3, coors);
+	}
+
+	public Triangle(Point a, Point b, Point c) {
+		super(3, new ArrayList<Point>());
+		super.addOneCoordinate(a.x, a.y);
+		super.addOneCoordinate(b.x, b.y);
+		super.addOneCoordinate(c.x, c.y);
+		
+	}
+	
+	@Override
+	public double area() {
+		// TODO Auto-generated method stub
+		double base = 0; 
+		return Math.abs((coordinates.get(0).x * 
+				(coordinates.get(1).y - coordinates.get(2).y) 
+				+ coordinates.get(1).x * (coordinates.get(2).y - 
+				coordinates.get(0).y) + coordinates.get(2).x * 
+				(coordinates.get(0).y - coordinates.get(1).y))/2);
+	}
+
+	@Override
+	public boolean isEquilateral() {
+		// TODO Auto-generated method stub
+		
+		return (distanceBetween(coordinates.get(0), coordinates.get(1)) == 
+				distanceBetween(coordinates.get(0), coordinates.get(2))
+				&& distanceBetween(coordinates.get(1), coordinates.get(2)) == 
+				distanceBetween(coordinates.get(0), coordinates.get(1)));
+	}
+	
+	private double distanceBetween(Point a, Point b) {
+		
+		return Math.sqrt((a.x -b.x) * (a.x -b.x) + 
+				(a.y -b.y) * (a.y -b.y));
```

Comment:
You probably need to deal with round off here.
check out this: https://stackoverflow.com/questions/4796746/double-value-to-round-up-in-java